core/lib/i18n/i18n.js

```diff
@@ -189,8 +189,27 @@ function createIcuMessageFn(filename, fileStrings) {
     const keyname = Object.keys(mergedStrings).find(key => mergedStrings[key] === message);
     if (!keyname) throw new Error(`Could not locate: ${message}`);
 
-    const filenameToLookup = keyname in fileStrings ? filename : getModulePath(import.meta);
-    const unixStyleFilename = path.relative(LH_ROOT, filenameToLookup).replace(/\\/g, '/');
+    let filenameToLookup;
+    if (path.isAbsolute(filename)) {
```

Comment:
Good catch. Yikes, we messed this up :(

Reply:
> Good catch. Yikes, we messed this up :(

Really this all only works because we manually set the stack-packs path to start with `node_modules/` :/ I assume publisher-ads message IDs don't work, but it doesn't appear they have anything but english strings for now, so at least it's not observable.

Really we should have thought through the lighthouse dependency vs peer dependency case. I can't think of a universally workable system except if everyone (including us) switched to a `package-name/path/to/file.js` for message IDs, though.

Not something we need to fix today, but we'll need to if e.g. pubAds wants to translate their strings, plugins take off, etc